python/shotgun_globals/cached_schema.py

```diff
@@ -920,3 +967,29 @@ def clear_cached_data(cls, project_id=None):
                 )
                 raise
 
+
+def _account_for_bubble_fields(sg_entity_type, field_name):
+    """Detect bubble fields and return the proper entity type and field name.
+
+    :param str sg_entity_type: The intput entity type name. If the field name
```

Comment:
Great! How about `shotgun.py`? then we can expose it as `sgtk.util.resolve_shotgun_bubble_fields()` or something like that - what do you think?